src/components/input-field/input-field.tsx

```diff
@@ -479,4 +480,9 @@ export class InputField {
             this.action.emit();
         }
     }
+    private handleWheel(event) {
+        if (this.type === 'number') {
+            event.target.blur();
```

Comment:
I think `preventDefault` would be better since it does not have the side effect of removing focus from the field
```suggestion
            event.preventDefault();
```